stdlib/tempfile.pyi

```diff
@@ -374,7 +374,7 @@ class SpooledTemporaryFile(IO[AnyStr], _SpooledTemporaryFileBase):
     def readlines(self, hint: int = ..., /) -> list[AnyStr]: ...  # type: ignore[override]
     def seek(self, offset: int, whence: int = ...) -> int: ...
     def tell(self) -> int: ...
-    def truncate(self, size: int | None = None) -> None: ...  # type: ignore[override]
+    def truncate(self, size: int | None = None) -> int: ...
```

Comment:
Thanks! It seems that this was implemented only in Python 3.11: https://github.com/python/cpython/blob/333c7dccd87c637d0b15cf81f9bbec28e39664fd/Lib/tempfile.py#L800

```suggestion
    if sys.version_info >= (3, 11):
        def truncate(self, size: int | None = None) -> int: ...
    else:
        def truncate(self, size: int | None = None) -> None: ... # type: ignore[override]
```

Reply:
Oh uh, good catch.